components/alert/test/alert.vdiff.js

```diff
@@ -41,4 +41,15 @@ describe('alert', () => {
 		const elem = await fixture(createAlertWithCloseButton(), { viewport: { width: 600 } });
 		await expect(elem).to.be.golden();
 	});
+
+	it('stacked', async() => {
```

Comment:
We have the multiple case covered for toasts but not for inline alerts. Toasts are positioned to provide some space between them. We'd probably want spacing between them, but unfortunately I don't think we have a good way to apply it within `d2l-alert`'s CSS since it's unaware the context/siblings - currently the consumer would have to apply the spacing. If we needed to enforce that, maybe we'd introduce a `d2l-alert-container` component to control the layout. Nevertheless, this seems like a worthwhile case to capture, even though it would seem that we're not really trying to provide anything special for it.